icu4j/main/classes/core/src/com/ibm/icu/message2/Mf2Parser.java

```diff
@@ -0,0 +1,758 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+package com.ibm.icu.message2;
+
+import java.util.Arrays;
+
+/**
+ * Class generated from EBNF.
+ *
+ * @internal ICU 72 technology preview
+ * @deprecated This API is for ICU internal use only.
+ */
+@Deprecated
+@SuppressWarnings("all") // Disable all warnings in the generated file
+class Mf2Parser
```

Comment:
Please remove the internal/deprecated/tech preview annotations+comments from non-public classes and methods.

Reply:
Done.